scripts/pre_push_hook.py

```diff
@@ -59,6 +59,12 @@
 
 
 class ChangedBranch(object):
+     """class for changing branch in which modified file is present.
+    
+    Attributes:
+      enter: for entering in the desired branch.
+      exit:  for coming out of current branch.
```

Comment:
Please remove the `Attributes:` section. In general, write only 1-2 lines summarising what a class is doing (in case of class docstrings only) and avoid any other information.

Reply:
resolved it.